### ISSUE TITLE
how to choose the validation set to evaluate the model, while using MPII dataset, without officail valid_set? 

### ISSUE THREAD

**Issue by ZhuohaoMai** [yuanyuanli85/Stacked_Hourglass_Network_Keras#1]

**Comment by yuanyuanli85** [comment]
The json file has mark the sample training or val . I got them from https://github.com/bearpaw/pytorch-pose. I did not double check if any difference with official release.
